### PR TITLE
BOCM-56

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

Templates, controller functions, validation pipes etc. for case officer journey check and confirm page + complete page

## Issue ticket number and link

https://pins-ds.atlassian.net/browse/BOCM-56

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

### CODE REVIEW

File: apps/web/src/server/views/app/layouts/app-wide.layout.njk [hunk 1]
```diff
@@ -1,3 +1,33 @@
 {% extends "app/layouts/app.layout.njk" %}
 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
 {% set containerClasses = 'pins-width-container' %}
+
+{% block content %}
```
Comment (brpins) [inline apps/web/src/server/views/app/layouts/app-wide.layout.njk:7]: I'm not sure why this change is needed – this extends from app.layout.njk, which is handling all of this anyway?

Comment (james-hill-kc) [reply apps/web/src/server/views/app/layouts/app-wide.layout.njk:7]: The `app.layout.njk` doesn't handle this, it's only in the derived templates such as `app-two-column.layout.njk`. Perhaps we should move it into the base layout instead?

Comment (brpins) [reply apps/web/src/server/views/app/layouts/app-wide.layout.njk:7]: Oh, right... I actually simplified the layouts in my new PR (this app-wide layout no longer exists) and it takes care of what you're doing here, so not sure if we should spend much time on this here. Yours should just work once both are merged and this file is removed.

**Review by radum — APPROVED** [review]

**Review by radum — APPROVED** [review]
